.github/quarkus-github-bot.yml

```
@@ -645,6 +645,12 @@ triage:
       directories:
         - extensions/vault/
         - integration-tests/vault
+    - id: netty
+      labels: [area/netty]
+      title: "netty"
+      notify: [franz1981, zakkak, cescoffier, jponge]
```

```suggestion
      notify: [franz1981, cescoffier, jponge]
```

I can only help with potential native mode issues and only in collaboration with someone familiar with netty so I am afraid I am not the right person to be pinged about such issues.

FWIW if an issue is labeled with `area/native-image` I get notified any way (regardless the extension).